tests/examples/externals/stencil.py

```diff
@@ -0,0 +1,74 @@
+from dusk.script import *
+
+# integration works naturally through python's import mechanism
+from definitions import *
+
+
+@stencil
+def horizontal_subdomains(edge1: Field[Edge, K]):
+
+    # examples of dusk syntax for horizontal subdomains
+
+    # short + slice syntax
+    with levels_upward, domain[nudging : halo - 1]:
+        edge1 = 1
+        # ...
+
+    # short + list syntax
+    with levels_upward, [nudging, halo - 1]:
+        edge1 = 1
+        # ...
+
+    # long + slice syntax
+    with levels_upward, domain[regions.nudging : regions.halo - 1]:
```

Comment:
I like this one the most, given that I would be able to swap `domain` and `levels_*`.